lib_expense/src/main/java/com/example/expenselogger/lib_expense/domain/contract/ExpenseContract.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.expenselogger.lib_expense.domain.contract
+
+import com.example.expenselogger.lib_expense.domain.model.Expense
+
+internal interface ExpenseContract {
+    fun saveExpenseEntry(expense: Expense)
+    fun getExpenseEntry(): Expense?
```

Comment:
When getting an expense item, You need to identify it somehow. Consider adding a unique id property to the expense object. Then you can use the id in this function as a parameter

Reply:
Yeaaa... skipped my mind. Thank youu